File: Problem/Easy/1501-1600/1588. Sum of All Odd Length Subarrays/_b.java
// 2 ms

class Solution {
	public int sumOddLengthSubarrays(int[] arr) {
		int sum = 0;

		for (int left = 0; left < arr.length; left++)
			for (int right = left + 1; right <= arr.length; right += 2)
				for (int i = left; i < right; i++)
					sum += arr[i];

		return sum;
	}
}

File: Problem/Easy/1501-1600/1588. Sum of All Odd Length Subarrays/b.py
class Solution:
    def sumOddLengthSubarrays(self, arr: List[int]) -> int:
        res = 0
        n = len(arr)

        for l in range(n):
            for r in range(l+1, n+1, 2):
                res += sum(arr[l:r])

        return res
